NetKAN/TweakScaleRescaled-SafetyNet.netkan

```diff
@@ -0,0 +1,17 @@
+spec_version: v1.18
+identifier: TweakScaleRescaled-SafetyNet
+name: TweakScale Rescaled SafetyNet
+abstract: >-
+  Attempts to prevent data loss when TweakScale
+  has been removed from an install.
+$kref: '#/ckan/spacedock/3514'
+$vref: '#/ckan/ksp-avc/GameData/TSSafetyNet/TweakScaleSafetyNet.version'
+release_status: stable
+license: CC-BY-NC-SA
+author:
+  - JonnyOThan
```

Comment:
This value of the author property would be pulled from SpaceDock automatically if you omitted it from the .netkan, which would be handy in case you add one or more co-authors in the future.